doc/BUILD.md

```diff
@@ -5,15 +5,16 @@ Quickstart
 ----------
 
 ### Steps
-First, install all ROS2 dependencies from the ROS2 Installation page: https://github.com/ros2/ros2/wiki/Linux-Development-Setup
+First, install all ROS2 dependencies from the ROS2 Installation page: https://index.ros.org/doc/ros2/Installation/Dashing/Linux-Development-Setup/
 
 Second, install the dependencies for this repository:
 ```sh
-RUN apt-get install -y \
-    libsdl-image1.2 \
-    libsdl-image1.2-dev \
-    libsdl1.2debian \
-    libsdl1.2-dev
+sudo apt-get install -y \
```

Comment:
We need to replace this with 'rosdep install' and all the proper flags. See what the Dockerfile does

Reply:
The initial build script already runs rosdep to install all the dependencies. If we want to replace this with rosdep, the build instructions will need to be changed a little bit.

Should we have instructions for two different build methods? such as:
method 1 (easy): run the script
method 2: create the workspaces, install dependencies, build packages, source install/setup.bash etc.

Reply:
I think that section can just be deleted, can't it? If the next step is running the initialize script, that'll call rosdep so if those are all keys then we're good

Reply:
Yes, it can be deleted. The script should be enough to build everything.